# Standard Header
std_msgs/Header header

# Array of `object_stamped` topics
zed_yolo_ros/Object[] objects
